runtimes/eden/src/lib.rs

```diff
@@ -381,6 +385,80 @@ sp_api::impl_runtime_apis! {
 
 			impl frame_system_benchmarking::Config for Runtime {}
 
+			use xcm::latest::prelude::*;
+			use frame_benchmarking::BenchmarkError;
+			use pallet_xcm_benchmarks::generic::Pallet as XcmGenericBenchmarks;
```

Comment:
Suggest adding the following code in higher level module (where it's accessible by for both add_benchmark and list_benchmark:
```
#[cfg(feature = "runtime-benchmarks")]
pub type XcmGenericBenchmarks = pallet_xcm_benchmarks::generic::Pallet<Runtime>;
pub type XcmFungibleBenchmarks = pallet_xcm_benchmarks::fungible::Pallet<Runtime>;
```
Then the line to add the type to the benchmark would look like:
```
list_benchmark!(list, extra, pallet_xcm_benchmarks::generic, XcmGenericBenchmarks);
```